nym-wallet/src/components/Bonding/modals/BondMoreModal.tsx

```diff
@@ -99,17 +114,6 @@ export const BondMoreModal = ({
           />
         </Box>
 
```

Comment:
In the CurrencyFormField component add `initialValue={additionalBond?.amount}`. Otherwise when you move to the confirm tx screen and then back again you will lose the value in the input field.

Reply:
I feel like bond more and unbond should be next to each other

![image](https://user-images.githubusercontent.com/13712075/205062899-4d44ba52-6365-4e11-941f-d324f558b54e.png)

Reply:
this order @fmtabbara It's how it is in Figma...